src/lib/profiles/data-management/Current/NotificationEngine.cpp

```diff
@@ -1761,6 +1762,10 @@ void NotificationEngine::Run()
     bool isClean  = true;
     bool isLocked = false;
 
+
+    VerifyOrExit(mNotifyTxEnabled, WeaveLogDetail(DataManagement, "Notify Transmission disabled. Enable to send Notifies");
```

Comment:
Consider to use WeaveLogProgress for this key message?

Reply:
Done.